### DISCUSSION TITLE
Add CSS File

### DISCUSSION THREAD

**Discussion by ROBOTIX44** [community/community#47020] [Codespaces]
### Select Topic Area

Product Feedback

### Body

Hi

I am new, I registred to test a simple css file but I can't, I made a new repository and when I ask for a codespace I can't choose any branch and any machine type. I have this errror :

A codespace cannot be created because no machine types are available. You may need to select a different branch, modify your container configuration, or adjust your organization's policy settings.

What is wrong ?

Sorry for my english^^

**Comment by Hangell** [comment]
Hi there! It looks like you're encountering an error when trying to create a codespace for your repository.

This error message suggests that there are no available machine types that meet the requirements for your repository's configuration. To resolve this issue, you can try one of the following:

Select a different branch: If you are trying to create a codespace for a specific branch, try choosing a different branch and see if that resolves the issue.

Modify your container configuration: Make sure that your repository's configuration is set up correctly and meets the requirements for creating a codespace.

Adjust your organization's policy settings: If you are a member of an organization, there may be policies in place that restrict the creation of codespaces. You may need to contact an organization administrator to modify these policies.

**Comment by ROBOTIX44** [comment]
Hi thank you for your reply. 

Here my html code :
`<!DOCTYPE html>
<html>
<head>
	<title>Ma boutique en ligne</title>
	
	<link rel="stylesheet" type="text/css" href="https://pastebin.com/raw/yQB0MKDW">
</head>
<body>
<p></p>

<p></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<div class="header-line"><a href="#">Notre boutique</a> <a href="#">Avis de nos clients</a> <a href="#">Contactez-nous</a></div>
<div class="services">
	<div>Livraison Rapide</div>
	<div>Livraison Internationale</div>
	<div>Les meilleurs prix</div>
	<div>Paiement s&eacute;curis&eacute;</div>
</div>
<div class="banner"><img src="#" alt="Banner Image" /></div>
<div class="menu-col">
	<h3>Cat&eacute;gories</h3>
	<ul class="doted">
		<li class="doted">Cat&eacute;gorie 1</li>
		<li class="doted">Cat&eacute;gorie 2</li>
		<li class="doted">Cat&eacute;gorie</li>
		<li class="doted">Nom du produit</li>
		<li class="doted">Prix</li>
		<li class="doted">Description</li>
	</ul>
	
	<table>
		<tbody>
			<tr>
				<th>Cat&eacute;gorie</th>
				<th>Nom du produit</th>
				<th>Prix</th>
				<th>Description</th>
			</tr>
			<tr>
				<td>V&ecirc;tements</td>
				<td>T-shirt</td>
				<td>20&euro;</td>
				<td>Un T-shirt confortable en coton</td>
			</tr>
			<tr>
				<td>V&ecirc;tements</td>
				<td>Jean</td>
				<td>50&euro;</td>
				<td>Un jean de qualit&eacute; sup&eacute;rieure</td>
			</tr>
			<tr>
				<td>Electronique</td>
				<td>Ordinateur portable</td>
				<td>1000&euro;</td>
				<td>Un ordinateur portable puissant</td>
			</tr>
		</tbody>
	</table>
	</div>
</body>
</html>`

and here my CSS code that I am trying to host here :

`body {
  font-family: Arial, sans-serif;
  margin: 0;
  padding: 0;
}

header {
  background-color: #333;
  color: #fff;
  display: flex;
  justify-content: space-between;
  padding: 20px;
}

header h1 {
  margin: 0;
}

header nav {
  display: flex;
}

header nav a {
  color: #fff;
  margin-left: 20px;
  text-decoration: none;
}

main {
  padding: 20px;
}

main h2 {
  margin-top: 50px;
}

footer {
  background-color: #333;
  color: #fff;
  display: flex;
  justify-content: center;
  padding: 20px;
}

@media (max-width: 600px) {
  header {
    flex-direction: column;
    align-items: center;
  }

  header h1 {
    margin-bottom: 20px;
  }

  header nav {
    margin-top: 20px;
    text-align: center;
  }

  main {
    padding: 10px;
  }
}
`

if you can take the time to see what's wrong that would be nice of you, i'm new to css and i want to learn...

**Comment by ROBOTIX44** [comment]
I don't know how to send you my html code here :) Maybe look attached it is ok
[aaaaaa.txt](https://github.com/community/community/files/10712581/aaaaaa.txt)
